admin/app/indexes/TagPages.scala

```diff
@@ -81,21 +81,25 @@ object TagPages {
     ("theguardian", "The Guardian"),
     ("theobserver", "The Observer"),
   )
-}
-
-class TagPages(implicit executionContext: ExecutionContext) extends GuLogging {
-
-  def alphaIndexKey(s: String): String = {
-    val badCharacters = """[^a-z0-9]+""".r
-
-    val maybeFirstChar = Try(badCharacters.replaceAllIn(asAscii(s).toLowerCase, "").charAt(0)).toOption
```

Comment:
The way this is done is a bit more work than is needed - the entire string has search and replace done with a regex to remove _bad_ characters- but all we want to find is the first _good_ character. This is now done in `alphaIndexCharacter()`, which returns an `Option[Char]`, populated if a good character can be found, and without having to mess around with `charAt(0)` throwing an `ArrayIndexOutOfBoundsException` and having to wrap that in a `Try`.